ngrinder-networkoverflow/src/main/java/org/ngrinder/network/NetworkOverFlow.java

```diff
@@ -66,23 +66,26 @@ public void startSampling(ISingleConsole singleConsole, PerfTest perfTest,
 			int consolePort = singleConsole.getConsolePort();
 			LOGGER.info("getLocalAgent count: {}, consolePort: {}", getLocalAgents().size(), consolePort);
 			int userSpecificAgentCount = 0;
+			int sharedAgentCount = 0;
 			for (AgentInfo each : getLocalAgents()) {
 				LOGGER.info("Agent name: {}", each.getName());
 				LOGGER.info("Agent region: {}, Agent port: {}", each.getRegion(), each.getPort());
-				if (each.getPort() == consolePort
-					&& StringUtils.contains(each.getRegion(), "owned")) {
-					LOGGER.info("userSpecific agent name: {}, userSpecific agent region: {}", each.getName(), each.getRegion());
-					userSpecificAgentCount++;
+				if (each.getPort() == consolePort) {
+					if (StringUtils.contains(each.getRegion(), "owned")) {
+						LOGGER.info("userSpecific agent name: {}, userSpecific agent region: {}", each.getName(), each.getRegion());
+						userSpecificAgentCount++;
+					} else {
+						sharedAgentCount++;
+					}
 				}
 			}
 			long configuredLimit = getLimit();
 			int totalAgentSize = allAttachedAgents.size();
-			int sharedAgent = (totalAgentSize - userSpecificAgentCount);
-			LOGGER.info("Plugin Info: totalAgentSize: {}, userSpecificAgentCount: {}", totalAgentSize, userSpecificAgentCount);
-			LOGGER.info("Plugin Info: configuredLimit: {}, sharedAgent: {}", configuredLimit, sharedAgent);
-			limit = sharedAgent == 0 ? Long.MAX_VALUE
-				: (long) (configuredLimit / (((float) sharedAgent) / totalAgentSize));
-
+			LOGGER.info("Plugin Info: totalAgentSize: {}", totalAgentSize);
+			LOGGER.info("Plugin Info: userSpecificAgentCount: {}, sharedAgentCount: {}", userSpecificAgentCount, sharedAgentCount);
+			limit = sharedAgentCount == 0 ? Long.MAX_VALUE
```

Comment:
This bug occurred because dedicated agent was not counted at the time of startSampling() call even though dedicated agent was used.

Conversely, user use 5 public(shared) agents for testing.
but at the time of call startSampling(), if sharedAgentCount is not 5, limit will be set to wrong value.

In the worst case, limit can be infinite, even though all agents are public.
it seems to be test whether these cases occur.

Reply:
This seems to cause a problem [in this part](https://github.com/naver/ngrinder-plugins/pull/2/commits/629a91bf64660065c28400461bafd64921a4cd7b#diff-80ba51d2ca9c3a2ee537da2a8c9eaf7bR73) when the agent port has not been updated yet.

Reply:
It could have caused a critical error. I reverted it and add retry policy